ksql-engine/src/main/java/io/confluent/ksql/KsqlEngine.java

```diff
@@ -82,8 +82,9 @@
 import io.confluent.ksql.util.Pair;
 import io.confluent.ksql.util.PersistentQueryMetadata;
 import io.confluent.ksql.util.QueryMetadata;
+import io.confluent.ksql.util.StreamsTopologyUtil;
 
-public class KsqlEngine implements Closeable, QueryTerminator {
+public class KsqlEngine implements Closeable {
```

Comment:
You've removed the implementation of `QueryTerminator` but you've left the unused interface around. Futher, this changes the behaviour of the `DropStream` and `DropTable` commands in that any queries against them are not dropped. Either this is not valid or i think these commands should check that all queries have been terminated before a corresponding `DropXxxx` Command is run